gnome-screenlock-password-opensuse132-20160620.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "type": "match",
+      "ypos": 237,
+      "height": 263,
+      "width": 458,
+      "xpos": 290
+    }
+  ],
+  "tags": [
+    "gnome-screenlock-password"

```

Comment:
err, what do mean "it is not implemented"? Isn't ENV-VERSION-13.2 automagically set based on VERSION in `vars.json`? somewhere? anywhere? magic?